docs/version0.48.md

```diff
@@ -42,6 +43,14 @@ From release 0.25.0 onwards, `zlibNX` hardware-accelerated data compression and
 
 For more information, see [`-XX:[+|-]UseZlibNX`](xxusezlibnx.md).
 
+### ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) JDWP support on Checkpoint/Restore In Userspace (CRIU) restore is enabled
+
+You can use the options that enable the JDWP support both on CRIU pre-checkpoint, and on restore as well.
+
+Also, a new parameter `suspendOnRestore`, specific to OpenJ9, is added to control the suspension of the Java&trade; application that must be debugged on restore. You can use the `suspendOnRestore=n` setting to enable debugging after restore even if no debugging is set pre-checkpoint.
```

Comment:
`suspendOnRestore` controls the suspension of JVM when the debug is enabled post-restore, it alone doesn't enable debugging.